cmake/CMakeLists.txt

```
@@ -624,6 +625,10 @@ if(PKG_ELECTRODE)
   target_link_libraries(lammps PRIVATE ${LAPACK_LIBRARIES})
 endif()
 
+if(PKG_ML-POD)
```

I would prefere:
```suggestion
if(PKG_ELECTRODE OR PKG_ML-POD)
```
and then dropping the duplication.